2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю lstm
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: invalid input syntax for type integer: "h"
LINE 4: ... timeframe = '1'                       AND sequence_id = 'h'
                                                                    ^

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю gru
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю lstm
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю gru
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю lstm
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю gru
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю lstm
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю gru
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 01:56:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 01:56:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 01:56:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю lstm
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: invalid input syntax for type integer: "h"
LINE 4: ... timeframe = '1'                       AND sequence_id = 'h'
                                                                    ^

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю gru
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю lstm
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю gru
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю lstm
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю gru
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю lstm
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю gru
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:01:01 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:01:01 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:01:01 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю lstm
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: invalid input syntax for type integer: "h"
LINE 4: ... timeframe = '1'                       AND sequence_id = 'h'
                                                                    ^

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю gru
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю lstm
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю gru
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для BTC: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю lstm
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю gru
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю lstm
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю gru
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка завантаження даних у data_loader для ETH: current transaction is aborted, commands ignored until end of transaction block

2025-05-25 02:07:13 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: 'NoneType' object is not subscriptable
2025-05-25 02:07:13 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:07:13 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'NoneType' object has no attribute 'copy'
2025-05-25 02:17:25 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 02:17:26 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor ініціалізовано для роботи з підготовленими даними
2025-05-25 02:17:26 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю lstm
2025-05-25 02:17:27 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 67848 записів для BTC-1h
2025-05-25 02:17:27 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:17:27 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:17:27 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:17:27 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:17:27 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 1h та моделлю gru
2025-05-25 02:17:29 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 67848 записів для BTC-1h
2025-05-25 02:17:29 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:17:29 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:17:29 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:17:29 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:17:29 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю lstm
2025-05-25 02:17:29 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 16980 записів для BTC-4h
2025-05-25 02:17:29 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:17:29 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:17:29 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:17:29 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:17:29 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для BTC з таймфреймом 4h та моделлю gru
2025-05-25 02:17:30 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 16980 записів для BTC-4h
2025-05-25 02:17:30 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:17:30 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:17:30 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:17:30 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:17:30 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю lstm
2025-05-25 02:17:31 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 67848 записів для ETH-1h
2025-05-25 02:17:31 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:17:31 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: list indices must be integers or slices, not list
2025-05-25 02:17:31 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:17:31 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'list' object has no attribute 'columns'
2025-05-25 02:17:31 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 1h та моделлю gru
2025-05-25 02:17:33 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 67848 записів для ETH-1h
2025-05-25 02:17:33 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:17:33 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: list indices must be integers or slices, not list
2025-05-25 02:17:33 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:17:33 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'list' object has no attribute 'columns'
2025-05-25 02:17:33 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю lstm
2025-05-25 02:17:33 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 16980 записів для ETH-4h
2025-05-25 02:17:33 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:17:33 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: list indices must be integers or slices, not list
2025-05-25 02:17:33 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:17:33 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'list' object has no attribute 'columns'
2025-05-25 02:17:33 - DataPreprocessor - INFO - [logger.py:103] - Підготовка завантажувача даних для ETH з таймфреймом 4h та моделлю gru
2025-05-25 02:17:34 - DataPreprocessor - INFO - [logger.py:103] - Успішно завантажено 16980 записів для ETH-4h
2025-05-25 02:17:34 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для ETH
2025-05-25 02:17:34 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для ETH: list indices must be integers or slices, not list
2025-05-25 02:17:34 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для ETH
2025-05-25 02:17:34 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для ETH: 'list' object has no attribute 'columns'
2025-05-25 02:24:34 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor инициализирован для работы с подготовленными данными
2025-05-25 02:24:35 - DataPreprocessor - INFO - [logger.py:103] - DataPreprocessor инициализирован для работы с подготовленными данными
2025-05-25 02:24:35 - DataPreprocessor - INFO - [logger.py:103] - Подготовка загрузчика данных для BTC с таймфреймом 1h и моделью lstm
2025-05-25 02:24:36 - DataPreprocessor - INFO - [logger.py:103] - Успешно загружено 67848 записей для BTC-1h
2025-05-25 02:24:36 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:24:36 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:24:36 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:24:36 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:24:36 - DataPreprocessor - INFO - [logger.py:103] - Подготовка загрузчика данных для BTC с таймфреймом 1h и моделью gru
2025-05-25 02:24:38 - DataPreprocessor - INFO - [logger.py:103] - Успешно загружено 67848 записей для BTC-1h
2025-05-25 02:24:38 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:24:38 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:24:38 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:24:38 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:24:38 - DataPreprocessor - INFO - [logger.py:103] - Подготовка загрузчика данных для BTC с таймфреймом 4h и моделью lstm
2025-05-25 02:24:38 - DataPreprocessor - INFO - [logger.py:103] - Успешно загружено 16980 записей для BTC-4h
2025-05-25 02:24:38 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:24:38 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:24:38 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:24:38 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:24:38 - DataPreprocessor - INFO - [logger.py:103] - Подготовка загрузчика данных для BTC с таймфреймом 4h и моделью gru
2025-05-25 02:24:39 - DataPreprocessor - INFO - [logger.py:103] - Успешно загружено 16980 записей для BTC-4h
2025-05-25 02:24:39 - DataPreprocessor - INFO - [logger.py:103] - Початок підготовки ознак для BTC
2025-05-25 02:24:39 - DataPreprocessor - ERROR - [logger.py:111] - Критична помилка при підготовці ознак для BTC: list indices must be integers or slices, not list
2025-05-25 02:24:39 - DataPreprocessor - WARNING - [logger.py:107] - Спроба створити базові ознаки для BTC
2025-05-25 02:24:39 - DataPreprocessor - ERROR - [logger.py:111] - Помилка fallback для BTC: 'list' object has no attribute 'columns'
2025-05-25 02:24:39 - DataPreprocessor - INFO - [logger.py:103] - Подготовка загрузчика данных для ETH с таймфреймом 1h и моделью lstm
